.travis.yml

```diff
@@ -73,6 +73,6 @@ jobs:
       # Currently not idempotent (https://github.com/lemurheavy/coveralls-public/issues/1435)
       script:
         - set -e
-        - pip install coveralls
+        - pip install coveralls==2.1.2
         - ln $COVERAGE_FILE .coverage
-        - coveralls
+        - coveralls --verbose
```

Comment:
Note:
Using the flag `debug` wouldn't publish the results back to the Github, but would show all check marks in green.
Using `--verbose` had the same logging output effect, with the GH report being honored.